src/components/ButtonsLanguages.jsx

```diff
@@ -0,0 +1,6 @@
+import './ButtonLanguage.css';
+
+function ButtonLanguage({ item }) {
+  return <button className="btn">{item}</button>;
+}
```

Comment:
We have standard prop name `children` for all cases like this. Use it please.

And it has to be passed to a component like `<MyBtn>My text</MyBtn>` not as a regular prop.